editoast/migrations/2025-11-19-092852-0000_modify_authn_user_to_handle_multiple_identities_for_same_user/down.sql

```diff
@@ -0,0 +1,18 @@
+alter table authn_user add column identity_id varchar(255);
+
+-- TODO PR: do we want to save one of the user identities on rollback ?
+--          should we pick it arbitrarily ?
```

Comment:
Picking arbitrarily on of it's identity seems fine. If it does not have an identity then we can skip the user.